### DISCUSSION TITLE
Optional Column tool tip feature

### DISCUSSION THREAD

**Discussion by mmani99** [bosskmk/pluto_grid#172] [Ideas]
Can we optionally enable tool tip for each column ? 
thanks

**Comment by bosskmk** [comment]
Features should be added.
Currently working on a null-safety version.
Once this is done, new features will be added to this version.
Is this a function you need before that?

**Reply by mmani99** [reply]
As suggested from others before, the most important feature would 
1)  cell background color api 
2) cell space (border width) to give a better look and feel
The column tool tip would be a nice feature. 
Thanks for the awesome work.

**Reply by bosskmk** [reply]
Thanks for your comment.
I work hard every time I have a little time on the weekends.
